SetDevCommandPrompt.cmd

```diff
@@ -1,12 +1,12 @@
 @echo off
 
-:: prefer building with Dev15
-
+:: Prefer building with Dev15 and try the simple route first (we may be running from a DevCmdPrompt already)
 set CommonToolsDir=%VS150COMNTOOLS%
 
-:: VS150COMNTOOLS is not set globally any more, so fall back to default preview location before trying Dev14
-if not exist "%CommonToolsDir%" set CommonToolsDir=%ProgramFiles(x86)%\\Microsoft Visual Studio\\VS15Preview\\Common7\\Tools\\
+:: VS150COMNTOOLS is not set globally any more, so fall back to using the managed query api before trying Dev14
+if not exist "%CommonToolsDir%" for /f "usebackq delims=" %%v in (`powershell -noprofile -executionPolicy Bypass -file "%RoslynRoot%build\scripts\locate-vs.ps1"`) do set "CommonToolsDir=%%v"
 if not exist "%CommonToolsDir%" set CommonToolsDir=%VS140COMNTOOLS%
 if not exist "%CommonToolsDir%" exit /b 1
 
 call "%CommonToolsDir%\VsDevCmd.bat"
```

Comment:
calling this sets my user directory to `C:\Users\jmarolf\Source` which i believe is the default directory to clone git repos to.  It would appear if anyone has ever done this on their machine it change their directory